TA-lansweeper-add-on-for-splunk/README.md

```diff
@@ -148,6 +148,10 @@ To uninstall app, user can follow below steps:
 
 RELEASE NOTES
 -------------
+Version 1.3.0 (Apr 2022)
+* Added support to collect antivirus details in both DBConenct and Modular inputs.
+* Added props/field-extraction for the Antivirus fields
+
```

Comment:
Upgrade Guide from Version 1.2.x to 1.3.0
* API input will collect the antivirus details from Lansweeper without any manual intervention.
* DB Connect input will require updating the SQL query of the input. Find the new query in inputs.conf.spec file.